config/locales/en.yml

```diff
@@ -267,6 +266,46 @@ en:
         look_up_link: "your Assignments page"
         footer: "You're receiving this email because you signed up for the %{title} challenge. For more information about this challenge and contact information for the moderators, please visit %{footer_link}."
         footer_link: "the challenge profile page"
+
+    invitation_to_claim:
+      subject: "[%{app_name}] Invitation to claim works"
+      greeting: "Hello from the %{app_name}!"
+      part5: "Claim or remove your works here"
```

Comment:
It looks like this has different punctuation for the two versions -- it should have a period for the HTML (which got removed altogether) and a colon for the text version (which is still in the view file). 

Since keeping the punctuation in the locale files rather than the view means creating separate versions anyway,  it might also be worthwhile to include the URL in the text version (e.g. `part5: Claim or remove your works here: %{claim_link}`) on the off chance translators need more control over where in the sentence it's displayed.